api_client/python/timesketch_api_client/sigma_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the Timesketch API client"""
+from __future__ import unicode_literals
+
+import unittest
+import mock
+
+# TODO remove config
+from . import test_lib
+from . import client
+
+
+
+class TimesketchSigmaTest(unittest.TestCase):
+    """Test Sigma."""
+
+    @mock.patch('requests.Session', test_lib.mock_session)
+    def setUp(self):
+        """Setup test case."""
+        self.api_client = client.TimesketchApi(
+            'http://127.0.0.1', 'test', 'test')
+        self.rule = self.api_client.get_sigma_rule(
+            "5266a592-b793-11ea-b3de-0242ac130004")
+        self.rules = self.api_client.list_sigma_rules()
```

Comment:
and same with this


`self.rule` is only used inside the `test_sigma_rule` function and `self.rules` only used in `test_sigma_rules` so why define them before each run?

also makes it easier to read if this is just on the same place where it is acually used

Reply:
yep, moved